ui/public/i18n/language.json

```
@@ -199,7 +199,7 @@
     "user_already_exists": "An account with the same name already exists",
     "user_not_found": "Account not found",
     "valid_username": "First character must be a letter, than only lower letters, numbers and symbols like '-' and '_'",
-    "valid_hostname_0_9223372036854775807": "Must be a valid hostname or a Fully qualified domain name"
+    "valid_hostname_0_9223372036854775807": "Must be a fully qualified domain name or a resource path component beginning with '/'"
```

```suggestion
    "valid_hostname_0_9223372036854775807": "Must be a fully qualified domain name or a path beginning with '/'"
```

---

It's not a "generic" path, it's a [resource path](https://en.wikipedia.org/wiki/URL#Syntax), just one _segment_. 

- `/A` is good
- `/A/B` is wrong

```suggestion
    "valid_hostname_0_9223372036854775807": "Must be a fully qualified domain name or a resource path segment beginning with '/'"
```